sql/dql/what-is-sql/what-is-an-rdbms.md

```diff
@@ -38,44 +38,27 @@ links:
 
 _Relational Databases_ store data in _tables_, which are similar to a spreadsheet in that they've got rows and columns. When you want data in one _table_ to reference data in another _table_, you create a _relation_ between them. A Relational Database Management System is therefore the program that actually does the creating, reading, updating, searching, filtering, and relating. It does this using a Structured Query Language called SQL.
 
-SQL has two main parts, the Query part, and the Definition part. DDL stands for Data Definition Language, and that is how we define data the tables contain.
-
-Here's an example of how a table can be created and defined:
-
-```sql
-CREATE TABLE users (
-  id INT,
-  name VARCHAR(500)
-);
-```
-
-This is executed on the *Database Server*, and the database server creates space for it.
-
-Then we could query it like this:
+SQL has two main elements, the Query part, and the Definition part. DDL stands for Data Definition Language, and that is how we define the data that the tables contain. The DDL commands are executed on the *Database Server*, and the database server creates space for it.
```

Comment:
actually, it has 3? 
DQL
DDL
DML (update delete) 

you can mention these with bullet points

Reply:
I know it has three but it our course we don't differentiate between DQL and DML. I'll do this change and whenever I encounter DML in the DQL topic I will mention that it is modification language.